nightmare_firmware/servos.cpp

```diff
@@ -68,7 +68,7 @@ void servo_rx_enb() {
 void servo::move(int ang){
 	servo_tx_enb();
   
-	ang = limit(map(ang, -120, 120, 0, 1000),0,1000); //angle comes in 2 bytes, 0..1000 means -120..120
+	ang = clamp(map(ang, -120, 120, 0, 1000),0,1000); //angle comes in 2 bytes, 0..1000 means -120..120
```

Comment:
Here (and line 88 too) you use `map` but in `utils.cpp` there's an `fmap`, which do you wanna use?

Reply:
also is the clamp necessary? You use it here because if ang is <-120 or >120 you get outside the range of 0..1000, but isn't it an error if it happens?

Reply:
If I remember correctly, the servos have a maximum angle of 240°, or -120 to 120, if you try to go above those limits it can result in instabilities or something

Reply:
So anyway, this is basically used just to be sure, so alright, cool